addons/actions/src/preset/addArgs.ts

```diff
@@ -19,12 +19,15 @@ export const inferActionsFromArgTypesRegex: ArgTypesEnhancer = (context) => {
   }
 
   const argTypesRegex = new RegExp(actions.argTypesRegex);
-  return mapValues(argTypes, (argType, name) => {
-    if (!argTypesRegex.test(name)) {
-      return argType;
+  return Object.entries<ArgTypes>(argTypes).reduce<ArgTypes>((acc, [name, argType]) => {
+    if (argTypesRegex.test(name)) {
+      acc[name] = { ...argType, defaultValue: action(name) };
+    } else {
+      acc[name] = argType;
     }
-    return { ...argType, defaultValue: action(name) };
-  });
+
+    return acc;
+  }, {});
```

Comment:
Do you think it's more important to remove this dependency from lodash, or have easily-identifiable code? Is there a way you could make these functions util/helper functions that are more easily recognized when invoked? These functions make my brain go through a lot of mental barriers to figure out what's going on at a glance.